approvals/src/main/java/no/sikt/nva/approvals/persistence/DynamoDbApprovalRepository.java

```diff
@@ -122,7 +122,7 @@ public Optional<Approval> findByIdentifier(NamedIdentifier namedIdentifier) thro
     }
```

Comment:
This was merged in another PR that I didn't review, but I can't help but comment anyway:

- Why do these methods catch any possible runtime exception?
- Why are these runtime errors rephrased as "could not find"? That is rather misleading, since an actual "no entity found" scenario returns `Optional.empty()`.